packages/cli/commands/preview.js

```diff
@@ -0,0 +1,72 @@
+const fs = require('fs');
+const path = require('path');
+const { i18n } = require('../lib/lang');
+const { logger } = require('@hubspot/cli-lib/logger');
+const { getAccountId } = require('../lib/commonOpts');
+const { EXIT_CODES } = require('../lib/enums/exitCodes');
+const { getCwd } = require('@hubspot/cli-lib/path');
+const { preview } = require('@hubspot/cli-lib/lib/preview');
+const { getUploadableFileList } = require('../lib/upload');
+const { trackCommandUsage } = require('../lib/usageTracking');
+
+const i18nKey = 'cli.commands.preview';
+
+exports.command = 'preview <src> <dest>';
+exports.describe = i18n(`${i18nKey}.describe`);
+
+exports.handler = async options => {
+  const { src, dest, notify, initialUpload } = options;
+
+  const accountId = getAccountId(options);
+  const absoluteSrc = path.resolve(getCwd(), src);
+
+  if (!validateSrcPath(absoluteSrc)) {
+    process.exit(EXIT_CODES.ERROR);
+  }
+
+  const filePaths = await getUploadableFileList(absoluteSrc, false);
+  trackCommandUsage('preview', accountId);
+  preview(accountId, absoluteSrc, dest, {
+    notify,
+    filePaths,
+    initialUpload,
+  });
+};
+
+exports.builder = yargs => {
+  yargs.positional('src', {
+    describe: i18n(`${i18nKey}.positionals.src.describe`),
+    type: 'string',
+  });
+  yargs.positional('dest', {
+    describe: i18n(`${i18nKey}.positionals.dest.describe`),
+    type: 'string',
+  });
+  yargs.option('initial-upload', {
+    alias: 'i',
+    describe: i18n(`${i18nKey}.options.initialUpload.describe`),
+    type: 'boolean',
+  });
+  return yargs;
+};
+
+const validateSrcPath = src => {
```

Comment:
Would love to see this function pushed above the builder and handler to follow the pattern in other commands -- or better, moved to the `cli-lib` in order to only have the handler and builder in this file (also similar to other commands)

Reply:
Looks like others do have their supporting functions toward the top - moved it there. 

I don't like moving it to `cli-lib` though, since it's only used before the `preview` process is called and not at all during it.

Maybe while adding support to VSCE we find we want that for both and it makes sense to just bake it into the command - but in the mean time a single helper function I don't think is a big deal. `hs watch` does some of the same logic.